{
  "configurations": [
    {
      "nodes": [
        {
          "cpeMatch": [
            {
              "criteria": "cpe:2.3:a:asustor:exfat_driver:1.0.0:r14:*:*:*:*:*:*",
              "matchCriteriaId": "90D17D19-2591-45EB-A250-4CE98A6CAD8D",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:asustor:exfat_driver:1.0.0:r15:*:*:*:*:*:*",
              "matchCriteriaId": "47ACBAA6-66D1-47F3-AE60-8768C48F7D9A",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:asustor:exfat_driver:1.0.0:r20:*:*:*:*:*:*",
              "matchCriteriaId": "1468FC2D-F6A0-457B-AB2F-0494D91F70CB",
              "vulnerable": true
            }
          ],
          "negate": false,
          "operator": "OR"
        }
      ]
    }
  ],
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "An issue was discovered in ASUSTOR exFAT Driver through 1.0.0.r20. When conducting license validation, exfat.cgi and exfatctl fail to properly validate server responses and pass unsanitized text to the system shell, resulting in code execution as root."
    },
    {
      "lang": "es",
      "value": "Se detect贸 un problema en ASUSTOR exFAT Driver versiones hasta 1.0.0.r20. Al dirigir la comprobaci贸n de licencia, exfat.cgi y exfatctl presenta un fallo al comprobar apropiadamente las respuestas del servidor y pasan texto no saneado hacia el shell del sistema, resultando en una ejecuci贸n del c贸digo como root."
    }
  ],
  "id": "CVE-2019-11689",
  "lastModified": "2020-08-24T17:37:01.140",
  "metrics": {
    "cvssMetricV2": [
      {
        "acInsufInfo": false,
        "baseSeverity": "HIGH",
        "cvssData": {
          "accessComplexity": "MEDIUM",
          "accessVector": "NETWORK",
          "authentication": "NONE",
          "availabilityImpact": "COMPLETE",
          "baseScore": 9.3,
          "confidentialityImpact": "COMPLETE",
          "integrityImpact": "COMPLETE",
          "vectorString": "AV:N/AC:M/Au:N/C:C/I:C/A:C",
          "version": "2.0"
        },
        "exploitabilityScore": 8.6,
        "impactScore": 10.0,
        "obtainAllPrivilege": false,
        "obtainOtherPrivilege": false,
        "obtainUserPrivilege": false,
        "source": "nvd@nist.gov",
        "type": "Primary",
        "userInteractionRequired": false
      }
    ],
    "cvssMetricV31": [
      {
        "cvssData": {
          "attackComplexity": "HIGH",
          "attackVector": "NETWORK",
          "availabilityImpact": "HIGH",
          "baseScore": 8.1,
          "baseSeverity": "HIGH",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "HIGH",
          "privilegesRequired": "NONE",
          "scope": "UNCHANGED",
          "userInteraction": "NONE",
          "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",
          "version": "3.1"
        },
        "exploitabilityScore": 2.2,
        "impactScore": 5.9,
        "source": "nvd@nist.gov",
        "type": "Primary"
      }
    ]
  },
  "published": "2020-03-18T15:15:11.797",
  "references": [
    {
      "source": "cve@mitre.org",
      "tags": [
        "Exploit",
        "Third Party Advisory"
      ],
      "url": "https://github.com/mikedamm/CVEs/blob/master/CVE-2019-11688.md"
    },
    {
      "source": "cve@mitre.org",
      "tags": [
        "Vendor Advisory"
      ],
      "url": "https://www.asustor.com/app_central/app_detail?id=776"
    }
  ],
  "sourceIdentifier": "cve@mitre.org",
  "vulnStatus": "Analyzed",
  "weaknesses": [
    {
      "description": [
        {
          "lang": "en",
          "value": "CWE-78"
        }
      ],
      "source": "nvd@nist.gov",
      "type": "Primary"
    }
  ]
}